### PR TITLE
redirect sourcefolders/inject source/dynamic sources

### PR DESCRIPTION
**Pull request by pascal-sauer**

### CODE REVIEW

**Comment by pascal-sauer** [comment]
I accidentally pushed the numeric_version clarifications to this PR, but it's really just minor changes ([a075200](https://github.com/pik-piam/madrat/pull/201/commits/a0752009d9003d1584fd78b1a1a8d18727506be6)), so it should be fine

File: R/getLinkFunction.R [hunk 1]
```diff
@@ -0,0 +1,41 @@
+getLinkFunction <- function() {
```
Comment (tscheypidi) [inline R/getLinkFunction.R:1]: please add a documentation

File: R/redirectSource.R [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+#'   # the following call will change directory
+#'   # into ~/TauExperiment instead of <getConfig("sourcefolder")>/Tau
+#'   readSource("Tau")
+#' }
+#' f()
+#' # Tau is only redirected in the local environment of f,
+#' # so it will use the usual source folder here
+#' readSource("Tau")
+#' }
+#' @export
```
Comment (tscheypidi) [inline R/redirectSource.R:31]: I find that confusing to have two rather similar functions exported (redirect and redirectSource). Lets have a discussion whether we can unify them somehow.

**Review by tscheypidi — CHANGES_REQUESTED** [review]

**Review by tscheypidi — APPROVED** [review]
looks good

**Pull request merged**